src/conversions.rs

```diff
@@ -180,7 +181,7 @@ fn clamp_to<D>(d: f64) -> D
 // https://heycam.github.io/webidl/#es-void
 impl ToJSValConvertible for () {
     #[inline]
-    unsafe fn to_jsval(&self, _cx: *mut JSContext, rval: MutableHandleValue) {
+    unsafe fn to_jsval(&self, _cx: *mut JSContext, mut rval: MutableHandleValue) {
```

Comment:
We should modify the ToJSValConvertible trait to accept a lifetime argument and use it for the rval argument.